internal/pkg/logging/logging.go

```diff
@@ -36,6 +40,43 @@ func (l *LogEntry) Fields() map[string]interface{} {
 	return l.logger.Data
 }
 
+// Convience Wrappers around logging actions
+
+// Info wraps the logrus Info function
+func Info(args ...interface{}) {
+	defaultLogger.logger.Info(args...)
+}
+
```

Comment:
I can see the difference between this set of logging wrappers and the below ones, but I'm a little bit unclear around the motivation here - do we need both sets of wrappers, or can they be merged together (with either solution around instantiating a new logger being the predominant one)?